tests/test-backend-ops.cpp

```diff
@@ -2064,20 +2064,22 @@ struct test_get_rows_back : public test_case {
 // GGML_OP_SET_ROWS
 struct test_set_rows : public test_case {
     const ggml_type type;
+    const ggml_type idx_type;
```

Comment:
This would be more consistent with the rest of the similar usages:

```suggestion
    const ggml_type type;
    const ggml_type type_idx;
```